packages/testnet-faucets/src/ts/request_queue.ts

```diff
@@ -46,9 +46,6 @@ export class RequestQueue {
             this._processNextRequestFireAndForgetAsync(recipientAddress);
         }, this._queueIntervalMs);
     }
-    protected _stop() {
-        clearInterval(this._queueIntervalId);
-    }
```

Comment:
Why is this removed? This seems unrelated to this PR.

Reply:
This is never called in the codebase. With that removed - we have a strong guarantee that `this._queueIntervalId` is never undefined and therefore can promise that to a compiler